package.json

```diff
@@ -25,7 +25,6 @@
     "test": "jest",
     "typescript": "tsc --noEmit",
     "lint": "eslint \"**/*.{js,ts,tsx}\"",
-    "prepare": "bob build",
```

Comment:
I feel like this was needed to build the TypeScript into JS or something. But it's one of those issues that I ran into a lot and every time I solved it I forgot to take notes on how I solved it.